x/stakeibc/keeper/hooks.go

```diff
@@ -205,16 +214,16 @@ func (k Keeper) StakeExistingDepositsOnHostZones(ctx sdk.Context, epochNumber in
 	})
 	for _, depositRecord := range stakeDepositRecords {
 		if depositRecord.DepositEpochNumber < cast.ToUint64(epochNumber) {
-			pstr := fmt.Sprintf("\t[STAKE] Processing deposit ID:{%d} DENOM:{%s} AMT:{%d}", depositRecord.Id, depositRecord.Denom, depositRecord.Amount)
+			pstr := fmt.Sprintf("\t[StakeExistingDepositsOnHostZones] Processing deposit ID:{%d} DENOM:{%s} AMT:{%d}", depositRecord.Id, depositRecord.Denom, depositRecord.Amount)
```

Comment:
It seems like there's a few instances where an object is printed with `%v`. Might be cleaner to do this of approach above instead where the individual keys are printed. Otherwise the keys wont be printed out by the object. 
For instance: https://github.com/Stride-Labs/stride/pull/167/files#diff-5e6dbf9edb5d12ad4a9f5aac34c45270bb006b965316c8b3852198d06b5692ffR299

Reply:
Good call - going to open a separate PR for this to unblock poolparty (I think we will still be able to tell fields apart with the current logging, it's just slightly more challenging)

Reply:
Yeah I think separate PR is good. You're right we can definitely infer which fields, but for the data structures with like 5+, it might be a little tedious to have to do that mapping mentally